Gems/PhysX/Code/Source/EditorHeightfieldColliderComponent.h

```diff
@@ -58,6 +63,20 @@ namespace PhysX
 
     private:
         AZ::u32 OnConfigurationChanged();
+        AZ::u32 OnToggleBakedHeightfield();
+        AZ::u32 GetBakedHeightfieldVisibilitySetting();
+
+        void RequestHeightfieldBaking() override;
+
+        // Utility functions for heightfield baking
+        void StartHeightfieldBakingJob();
+        void FinishHeightfieldBakingJob();
+        bool CheckHeightfieldPathExists();
```

Comment:
maybe `CheckHeightfieldPathExists` could be a const function?

Reply:
We clear the path if it's not valid anymore, hence made as non-const. It's a private utility function, so not expected to be called externally.